source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,56 @@
+RoboRIO2 Imaging
+================
+
+The NI roboRIO 2.0 boots from a microSD card configured with an appropriate boot image containing the NI Linux Real-Time OS, drivers, and libraries specific to FRC. There are two ways to create the SD card -- with the roboRIO using the imaging tool, or with a laptop and an SD burner application.
```

Comment:
Will the imaging tool work with a blank SD card? I assume it won't, but this seems to imply that it would.

Reply:
I don't know how you got that.  It says "boots from a microSD card configured with an appropriate boot image" blank would not be appropriate.

Reply:
It says two ways to create the SD card. To me that implies from scratch.

Reply:
It then lists the two ways which are both using an image and an image tool.  I am still not seeing the confusion.